README.md

```diff
@@ -106,6 +106,13 @@ Other commands while in pomodoro mode:
 
 ```
 
+To change default pomodoro and break durations set following variables (these should be numbers):
```

Comment:
I think this is great! Might be good to just let people know if it is in seconds/minutes/hours when setting them.